Gemfile

```diff
@@ -17,6 +17,7 @@ gem 'pg', '0.18.3'
 
 gem 'nokogiri', '~> 1.8.0'
 gem 'rails', '~> 5.1'
+gem 'thor'
```

Comment:
> Let's specify the version?

Not sure what the best practice here is. Because Thor is already a required gem for railties. Which probably already specifies the version somehow. 
So not really sure what is reasonable to set here.